src/wh_client_cryptocb.c

```diff
@@ -244,7 +250,9 @@ int wolfHSM_CryptoCb(int devId, wc_CryptoInfo* info, void* inCtx)
             break;
 #endif /* HAVE_AESGCM */
 #endif /* NO_AES */
+#ifdef HAVE_AES
```

Comment:
don't protect the control flow constructs, see above

Reply:
When using the macros guard it should be `#ifndef NO_AES`?